src/components/BookForm.js

```diff
@@ -11,26 +11,25 @@ const BookForm = ({ addBook }) => {
     const author = authorRef.current.value;
     if (title && author) {
       addBook({ title, author });
+      titleRef.current.value = '';
+      authorRef.current.value = '';
     }
-    titleRef.current.value = '';
-    authorRef.current.value = '';
   };
+
   return (
     <div>
       <form onSubmit={submitBookHandler}>
         <div>
           <label htmlFor="title">
             <p>Title:</p>
             <input type="text" id="title" ref={titleRef} />
-            {' '}
           </label>
         </div>
         <div>
           <label htmlFor="author">
             <p>Author:</p>
             <input type="text" id="author" ref={authorRef} />
           </label>
-
         </div>
         <button type="submit">Add</button>
```

Comment:
- [ ] kindly consider  including an event handler to **add** a book to the `books` array (with attributes *title* and *author*) to meet the below requirement 
> Add a <Button> component, which includes:
> - An event handler that adds a book to the books array (with attributes id, title, and author)